src/EditorFeatures/Core/Implementation/ExtractMethod/AbstractExtractMethodCommandHandler.cs

```diff
@@ -142,9 +142,11 @@ public void ExecuteCommand(ExtractMethodCommandArgs args, Action nextHandler)
                     if (notificationService != null)
                     {
                         if (!notificationService.ConfirmMessageBox(
+                                EditorFeaturesResources.ExtractMethodFailedReasons + Environment.NewLine + Environment.NewLine +
                                 string.Join(Environment.NewLine, result.Reasons) + Environment.NewLine + Environment.NewLine +
```

Comment:
Nit: Not sure if this intentional, but notice 3 Newlines after `ExtractMethodFailedReasons` and 2 Newlines after listing the actual reasons.. If it looks alright in the UI, nevermind.